service/controller/v15/templates/cloudformation/guest/auto_scaling_group.go

```diff
@@ -5,22 +5,23 @@ const AutoScalingGroup = `{{define "autoscaling_group"}}
   {{ $v.ASGType }}AutoScalingGroup:
     Type: "AWS::AutoScaling::AutoScalingGroup"
     Properties:
-      VPCZoneIdentifier:
-        - !Ref PrivateSubnet
       AvailabilityZones: [{{ $v.WorkerAZ }}]
       DesiredCapacity: {{ $v.ASGMinSize }}
-      MinSize: {{ $v.ASGMinSize }}
+      HealthCheckGracePeriod: {{ $v.HealthCheckGracePeriod }}
+      HealthCheckType: ELB
```

Comment:
This line is added. The rest is just reordered alphabetically.

Reply:
Still have to test this a bit manually and see how the updates behave with this.